typescript/cli/src/context/context.ts

```diff
@@ -134,7 +134,7 @@ export async function getDryRunContext(
  * and an override one (such as a local directory)
  * @returns a new MergedRegistry
  */
-function getRegistry(
+export function getRegistry(
```

Comment:
I see you're using this in a test helper. I'm _slightly_ concerned someone else will see this is exported and think it's the way to get a registry object (instead of via the command context). Could you have the test get the registry via the already exported `getContext` fn? If not np

Reply:
Fixed to use getContext